tests/core/daemon/test_daemon.py

```diff
@@ -19,10 +19,6 @@
 
 
 class TestDaemon:
-    @pytest_asyncio.fixture(scope="function")
-    async def get_daemon(self, get_b_tools):
```

Comment:
the daemon is started by `setup_full_system()` now (called by the `simulation` fixture below)